decidim-core/lib/decidim/exporters/csv.rb

```diff
@@ -15,8 +15,8 @@ class CSV < Exporter
       # provided serializer and following the previously described strategy.
       #
       # Returns an ExportData instance.
-      def export
-        data = ::CSV.generate(headers: headers, write_headers: true, col_sep: ";") do |csv|
+      def export(col_sep = ";")
```

Comment:
@aitorlb It would be great to have the possibility to set the default separator from the initializer

Reply:
I agree with @agustibr . Do you, @decidim/lot-core ?